pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -710,3 +710,17 @@ func (am *Alertmanager) timeoutFunc(d time.Duration) time.Duration {
 type nilLimits struct{}
 
 func (n nilLimits) MaxNumberOfAggregationGroups() int { return 0 }
+
+// cleanUp removes the silences and notification-log from kv store/disk used by this
+// alertmanager.
+func (am *Alertmanager) cleanUp() {
+	if err := am.fileStore.Delete(context.Background(), silencesFilename); err != nil {
+		am.logger.Warn("failed to delete file from filestore", "file", silencesFilename)
+	}
+	if err := am.fileStore.Delete(context.Background(), notificationLogFilename); err != nil {
+		am.logger.Warn("failed to delete file from filestore", "file", notificationLogFilename)
+	}
+	if err := os.RemoveAll(am.WorkingDirPath()); err != nil {
+		am.logger.Warn("failed to cleanup resources from filesystem", "err", err)
+	}
```

Comment:
WorkingDirPath contains the files above. I think RemoveAll is sufficient here. Probably encapsulate it to FileStore's method CleanUp or something

Reply:
The files above can be additionally in the KV-Store database (which is just a sql table), so we have to do both, remove them from the KV-Store database by calling `Delete` and remove the organization directory.